xqueue_watcher/client.py

```diff
@@ -184,9 +189,16 @@ def run(self):
                         num_tries)
                 else:
                     break
+        poll_interval = self.poll_interval
         while self.running:
             if not self.process_one():
-                time.sleep(self.poll_interval)
+                jitter = random.random() * 3
```

Comment:
To avoid having multiple threads hitting the server at exactly the same time. Our remote config process [does a similar thing](https://github.com/edx/hermes/blob/master/hermes.py#L93).

Reply:
:ok_hand: